src/newgrf.cpp

```diff
@@ -7093,6 +7108,18 @@ static void SpriteReplace(ByteReader *buf)
 			i, num_sprites, first_sprite
 		);
 
+		if (first_sprite + num_sprites >= SPR_OPENTTD_BASE) {
+			/* Outside allowed range, check for GRM sprite reservations. */
+			if (!IsGRMReservedSprite(first_sprite, num_sprites)) {
+				GrfMsg(0, "SpriteReplace: Set {}] Changing {} sprites, beginning with {}, above limit of {} and not reserved, ignoring.",
```

Comment:
I don't know what the intention of the (unopened) square bracket is, but it's inconsistent. Not sure whether the following is more along the lines of what's wanted though.
```suggestion
				GrfMsg(0, "SpriteReplace: Set {} changing {} sprites, beginning with {}, above limit of {} and not reserved, ignoring.",
```

Reply:
Oops, it's meant to be `[Set {}]` like the message on line 7107.